flagscale/auto_tuner/tuner.py

```diff
@@ -78,16 +78,23 @@ def __init__(self, config: DictConfig):
                 self.config.experiment.runner.nnodes = int(os.environ["AIRS_SIZE"])
 
             if os.environ.get("AIRS_ACCELERATOR_COUNT", None):
-                self.config.experiment.auto_tuner.nproc_per_node = int(
-                    os.environ["AIRS_ACCELERATOR_COUNT"]
+                # Set config
+                self.config.experiment.auto_tuner.nproc_per_node = (
+                    int(os.environ["AIRS_ACCELERATOR_COUNT"]) * 2
+                    if "luvatar_BI" in os.environ["AIRS_ACCELERATOR_MODEL"]
+                    else int(os.environ["AIRS_ACCELERATOR_COUNT"])
                 )
                 # Set original config
-                self.orig_config.experiment.runner.nproc_per_node = int(
-                    os.environ["AIRS_ACCELERATOR_COUNT"]
+                self.orig_config.experiment.runner.nproc_per_node = (
+                    int(os.environ["AIRS_ACCELERATOR_COUNT"]) * 2
+                    if "luvatar_BI" in os.environ["AIRS_ACCELERATOR_MODEL"]
+                    else int(os.environ["AIRS_ACCELERATOR_COUNT"])
                 )
                 # Set config
-                self.config.experiment.runner.nproc_per_node = int(
-                    os.environ["AIRS_ACCELERATOR_COUNT"]
+                self.config.experiment.runner.nproc_per_node = (
+                    int(os.environ["AIRS_ACCELERATOR_COUNT"]) * 2
+                    if "luvatar_BI" in os.environ["AIRS_ACCELERATOR_MODEL"]
+                    else int(os.environ["AIRS_ACCELERATOR_COUNT"])
```

Comment:
Is there any way to move these platform related code into a standalone place? We may support differnt cloud platform.

Reply:
The platform code has been removed to platform.py and different platforms code will be in this file.